packages/runtime-core/src/rendererTemplateRef.ts

```diff
@@ -65,15 +65,12 @@ export function setRef(
   const refs = owner.refs === EMPTY_OBJ ? (owner.refs = {}) : owner.refs
   const setupState = owner.setupState
   const rawSetupState = toRaw(setupState)
-  const canSetSetupRef =
-    setupState === EMPTY_OBJ
-      ? () => false
-      : (key: string) => {
-          if (__DEV__ && knownTemplateRefs.has(rawSetupState[key] as any)) {
-            return false
-          }
-          return hasOwn(rawSetupState, key)
-        }
+  const canSetSetupRef = (key: string) => {
```

Comment:
I don’t think this change is reasonable. The original logic was very clear: if `setupState` is empty, it means doing nothing. With this PR, the development environment checks `knownTemplateRefs.has` every time, regardless of whether `setupState` empty or not, which is obviously unreasonable.

Reply:
We don't even need to check it? Or may need to supplement test cases